### ISSUE TITLE
Exported view body contains references to previous database

### ISSUE THREAD

**Issue by sergiokessler** [HeidiSQL/HeidiSQL#1336]
**Preconditions**
* HeidiSQL version: 11.2.0.6313
* Database type and version: Mysql 8.0.18
* OS: Linux

**Describe the bug**

- you have database A, where there is a view called `my_view` (`A.my_view`) that uses `table1`
- the definition of `my_view` in A does NOT prefix `table1` with the db_name
- you copy entire db A to db B, using "Export database as SQL"
- you found (after mucho head banging) that `B.my_view` uses `A.table1`

**To Reproduce**

follow the step above...

**Screenshots or Crash reports**

N/A

**Comment by ansgarbecker** [comment]
Are you sure the stored view code does not reference the old "A" database? I mean, HeidiSQL tries to present you the view code you entered once you created it, but the code is probably different on the server. Please find the view code using such a query:
```
SHOW CREATE VIEW my_view;
```

**Comment by sergiokessler** [comment]
yup, sure, it was the first thing I checked before submitting this issue...

I also tried that command:
```
SHOW CREATE VIEW producto_full;
```
to see the original view in db A, just now, and the tables are not prefixed...

**Comment by ansgarbecker** [comment]
I did again try to reproduce on MySQL 8.0.33, creating a simple table and a view in an empty database A:

```sql
CREATE TABLE `table2` (
    `id` INT NULL
)
COLLATE='utf8mb4_unicode_ci';
CREATE ALGORITHM = UNDEFINED SQL SECURITY DEFINER
    VIEW `view2` AS SELECT * FROM table2;
```

After saving changes, `SHOW CREATE VIEW view2` shows the following, server-side manipulated code:

```sql
CREATE ALGORITHM=UNDEFINED DEFINER=`root`@`localhost` SQL SECURITY DEFINER
    VIEW `view2` AS select `table2`.`id` AS `id` from `table2`
```

After exporting the table and view to a new empty database "test3", the new view code changed again, now with references to the old "test2" database:

```sql
CREATE ALGORITHM=UNDEFINED DEFINER=`root`@`localhost` SQL SECURITY DEFINER
    VIEW `view2` AS select `test2`.`table2`.`id` AS `id` from `test2`.`table2`
```
